view/src/storage.rs

```diff
@@ -739,47 +836,8 @@ impl Storage {
             let position = (u64::from(note_record.position)) as i64;
             let source = note_record.source.to_bytes().to_vec();
 
-            // If this note corresponded to a previously quarantined note, delete it from quarantine
```

Comment:
Why is all this code removed in this PR? It seems to break the smoke tests

Reply:
It looks like it reverts the note commitment fix added in #1520 so could just be due to a rebase, I think we do want that logic (the smoke test in question is failing as it is a regression test for the bug in #1520)